pages/your_tasks/dm_coordination.md

```diff
@@ -0,0 +1,89 @@
+  * Ensuring that the strategy for documentation and data exchange is adequate.
+  * Planning for long-term storage and data sharing for all partners.
+* Try to keep it simple and find a balance between the amount of necessary details and not making documentation overly difficult and discouraging.
+
+## How to sustain and monitor data management in collaborative projects?
+
+### Description
+If you want to secure a constant effort on research data management across the whole project and after the project, with stable data quality, this requires dedicated monitoring and focus from the project management and the DMWG.
+
+### Considerations
+* Be aware of a possible lack of storage infrastructure, thus requiring financing that might not be part of the initially proposed project budget.
+* Monitor that data is not lost, e.g. by missing or not-implemented backup procedures. 
+* Monitor that metadata is collected, and organized according to agreed standards and that no discrepancies between metadata and reality exist (e.g. sample ID mismatches).
+* Copyright licenses connected to the reuse of the project outputs should be applicable long-term beyond the project's lifetime.
+* Some data might need to be made available through restricted access mechanisms, e.g. for sensitive (personal) data. Safeguarding the functionality of a data access committee beyond the project period requires coordination on an institutional level.
+* The quality, size, location, and reuse of datasets might be an object of project reporting and be used as a performance indicator. 
+
+### Solutions
+* For long-term preservation of valuable project outcomes, try to look for additional funding within your organization or from other funders, or create a feasible strategy to use e.g. institutional, national, or international repositories, with their own preservation policy.
+* Automate backups as much as possible with your IT department and test them. Backup log files should be actively monitored for potential issues.
+* The DMWG should perform regular checks with data hosting project partners to check that data and metadata are produced according to the DMP, and that agreed backup routines are followed. 
+* Post-project preservation of data should consider reproducibility/replicability of the data from a long-term perspective, as well as the potential reuse value. You might want to consult a [data preservation](preserving) service (e.g. national data centres) regarding larger amounts of data. As an initial inspiration, you can start from [this data value checklist](https://www.bgs.ac.uk/download/ngdc-data-value-checklist/) and adapt it for your legal framework and domain.
+* Do not leave the data holders with an impossible task of trying to figure out who can have access to the resources, but rather use a license that allows re-use of data - e.g. some projects (project call level) demand to use CC-BY-4.0 and refrain from using bespoke, non-standard license terms.
```

Comment:
what does (project call level) refer to? 
or did you mean to say "(depending on activity level)"?

Reply:
I think it means that some funder or specific type of project require CC-BY-4.0 licence.